grails-app/views/doiResolve/_phylolink.gsp

```diff
@@ -0,0 +1,218 @@
+<%@ page contentType="text/html;charset=UTF-8" %>

+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="layout" content="main"/>
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+    %{-- Google Analytics --}%
+    <script>
+        window.ga = window.ga || function () {
+                    (ga.q = ga.q || []).push(arguments)
+                };
+        ga.l = +new Date;
+        ga('create', '${grailsApplication.config.googleAnalyticsId}', 'auto');
+        ga('send', 'pageview');
+    </script>
+    <script async src='//www.google-analytics.com/analytics.js'></script>
+    %{--End Google Analytics--}%

```

Comment:
Has this been tested with the ALA footer that includes GA codes itself? I have a feeling it will cause a JS error, having 2 sets of codes/includes.